execution_engine/src/storage/trie_store/lmdb.rs

```diff
@@ -219,9 +219,9 @@ impl ScratchTrieStore {
                 continue;
             };
 
-            let lazy_trie: LazyTrieLeaf<Key, StoredValue> =
-                trie::lazy_trie_deserialize(trie_bytes.clone())?;
-            tries_to_write.extend(trie::lazy_trie_iter_children(&lazy_trie));
+            let lazy_trie: LazilyDeserializedTrie =
+                bytesrepr::deserialize(trie_bytes.clone().into())?;
```

Comment:
Not sure if this will compile but worth a try:

```suggestion
                bytesrepr::deserialize_from_slice(&trie_bytes)?;
```

Reply:
fixed in https://github.com/casper-network/casper-node/pull/4083/commits/42c4d61a9a2fea9d299d65b128d3854d869f1870